SwiftyContextMenu/ContextMenuActionTableViewCell.swift

```diff
@@ -8,25 +8,86 @@
 import UIKit
 
 class ContextMenuActionTableViewCell: UITableViewCell {
-
+    
     private var rightImageView = UIImageView(frame: CGRect(x: 0, y: 0, width: 25, height: 25))
-
+    private var lightSelectedBackgroundView: UIVisualEffectView!
+    private var darkSelectedBackgroundView: UIView!
+    private var separatorView: ContextMenuSeparatorView!
+    private var style: ContextMenuUserInterfaceStyle = .light {
+        didSet {
+            updateSelectedBackgroundView()
+            separatorView.style = style
+        }
+    }
+    
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
         backgroundColor = .clear
         rightImageView.contentMode = .scaleAspectFit
         accessoryView = rightImageView
+        
+        prepareSelectedBackgroundView()
+        
+        separatorView = ContextMenuSeparatorView(frame: CGRect(x: 0, y: 0, width: bounds.width, height: 0.33), style: self.style)
+        addSubview(separatorView)
     }
-
+    
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-
-    func configure(action: ContextMenuAction) {
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        updateSelectedBackgroundView()
+    }
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        updateSelectedBackgroundView()
+    }
+    
+    func configure(action: ContextMenuAction, with style: ContextMenuUserInterfaceStyle) {
         textLabel?.text = action.title
         textLabel?.numberOfLines = 0
-        textLabel?.textColor = action.tintColor
         rightImageView.image = action.image?.withRenderingMode(.alwaysTemplate)
-        rightImageView.tintColor = action.tintColor
+        
+        self.style = style
+        switch style {
+        case .automatic:
+            textLabel?.textColor = action.tintColor
+            rightImageView.tintColor = action.tintColor
+        case .light:
+            textLabel?.textColor = action.tintColor
+            rightImageView.tintColor = action.tintColor
+        case .dark:
+            textLabel?.textColor = action.tintColorDark
+            rightImageView.tintColor = action.tintColorDark
+        }
+    }
+    
+    fileprivate func prepareSelectedBackgroundView() {
```

Comment:
this is more an "add" than a "prepare", right? plus I'm not sure why this is `fileprivate` instead of `private`.
I would change this into: `private func addSelectedBackgroundView()`